core/controllers/suggestion.py

```diff
@@ -92,7 +92,11 @@ class FrontendBaseSuggestionDict(TypedDict):
         str,
         Dict[
             str,
-            Optional[Dict[str, Union[str, List[Dict[str, Union[str, int]]]]]]
+            Union[
+                Optional[
+                    Dict[str, Union[str, List[Dict[str, Union[str, int]]]]]],
+                List[str]
```

Comment:
Could you explain why this was changed? I don't see a List[str] added to the handler.